### PR TITLE
Runtime CR examples updated

### PR DESCRIPTION
**Pull request by akgalwas**
**Description**

Changes proposed in this pull request:

- Exemplary Runtime CR yamls fixed
- Status condition contains detailed error message from Gardener (temporary solution, in the future we must provide error category in the CR's status)

**Related issue(s)**
#332

### CODE REVIEW

File: docs/adr/assets/runtime-examples/azure-fremium.yaml [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
 spec:
   shoot:
     # spec.shoot.name is set required
-    name: shoot-name
+    name: azure-fremium
     # spec.shoot.purpose is required
     purpose: evaluation
     # spec.shoot.region is required
-    region: eu-central-1
+    region: polandcentral
```
Comment (Disper) [inline docs/adr/assets/runtime-examples/azure-fremium.yaml:24]: what was your intention with this change?

File: docs/adr/assets/runtime-examples/azure-lite.yaml [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
     kyma-project.io/region: region
     operator.kyma-project.io/kyma-name: kymaName
-  name: runtime-id
+  name: azure-lite
   namespace: kcp-system
 spec:
   shoot:
     # spec.shoot.name is required
-    name: shoot-name
+    name: azure-lite
```
Comment (Disper) [inline docs/adr/assets/runtime-examples/azure-lite.yaml:20]: `usernameClaim: sub` at line 40 has incorrect indentation and can't be applied.

**Review by Disper — COMMENTED** [review]
Azure, GCP and CCEE contains `additionalOidcConfig:` sections while other examples do not. Is it intentional?

**Comment by Disper** [comment]
I've managed to test succesfully the examples for 
- `aws-freemium`
- `azure-lite` ( I had to do [this change](https://github.com/kyma-project/infrastructure-manager/pull/345#discussion_r1724512485) in order to make it work)
- `gcp`

File: docs/adr/assets/runtime-examples/azure-lite.yaml (hunk 2 above)
Comment (akgalwas) [reply docs/adr/assets/runtime-examples/azure-lite.yaml:20]: > `usernameClaim: sub` at line 40 has incorrect indentation and can't be applied.

Fixed

**Comment by akgalwas** [comment]
> Azure, GCP and CCEE contains `additionalOidcConfig:` sections while other examples do not. Is it intentional?

The following yamls : `aws`, `azure`, `gcp`, `sap-converged-cloud` are supposed to contain all the fields that KEB can specify. Examples prefixed with `trial`, `freemium`, `lite`, or `minimal` are supposed to be smaller so they doesn't contain `additionalOidcConfig` section on purpose.

**Review by Disper — APPROVED** [review]

**Pull request merged**